src/types/logging/Transformers.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * The logform package exports `TransformableInfo` but it sets the
+ * message type to `any`. Here we take that exact type and recreate it
+ * to have more specific type info.
+ */
+export interface ITransformableInfo {
+	level: string;
+	message: string;
+	[key: string]: string;
+}
```

Comment:
Same as in `index.ts` : 
```suggestion
// Copyright 2017-2022 Parity Technologies (UK) Ltd.
// This file is part of Substrate API Sidecar.
//
// Substrate API Sidecar is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License
// along with this program.  If not, see <http://www.gnu.org/licenses/>.

/**
 * The logform package exports `TransformableInfo` but it sets the
 * message type to `any`. Here we take that exact type and recreate it
 * to have more specific type info.
 */
export interface ITransformableInfo {
	level: string;
	message: string;
	[key: string]: string;
}
```

Reply:
Good catch!